quantum/keymap_extras/sendstring_workman_zxcvm.h

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2018 Jacob Jerrell
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Sendstring lookup tables for Workman ZXCVM layout - defaults CVM 
+ * keys to their QWERTY positions to make this layout easier to use with
+ * other other QWERTY-like layouts installed on the same machine 
+ * (mainly because of CTRL+X, CTRL+C and CTRL+V shortcuts).
+ */
+
+#pragma once
+
+#include "keymap_workman_zxcvm.h"
+
+// clang-format off
+
+const uint8_t ascii_to_keycode_lut[128] PROGMEM = {
```

Comment:
You will also need to add an `ascii_to_shift_lut` lookup table for shifted characters such as `%`, `<`, `>` etc.

Reply:
I notice it's missing from quite a few custom layouts and especially from the original workman layout. Is there any good examples / documentation to follow on how to do this?

Reply:
`ascii_to_keycode_lut` will not work correctly without `ascii_to_shift_lut`, so ideally it should be added wherever it's missing.

The idea is that `ascii_to_shift_lut` should have a 1 wherever Shift needs to be pressed in order to produce the corresponding ASCII character, and a 0 otherwise. For example, `%` in the case of your Workman layout would be produced by pressing `WK_COMM` while holding Shift, so your `ascii_to_shift_lut` should have a 1 in the `%` position.

Take a look at the default US ANSI `ascii_to_shift_lut` for reference:

https://github.com/qmk/qmk_firmware/blob/1c3c9b927190d4a5ea8365f01a336d5bdd241e21/quantum/quantum.c#L340-L361

Reply:
This doesn't actually need anything other than `ascii_to_keycode_lut`, since it looks nearly identical to the existing Workman keymap aside from placement of a couple alphas, so we can simply use the default shift and altgr LUTs.

Reply:
You are correct, my mistake. I thought Workman also swapped some of the punctuation characters around, but it looks like it doesn't. So this will be fine as is.

Feel free to mark this as resolved.

Reply:
In fact... the sendstring header is completely identical, you could probably just drop it altogether and use the regular `sendstring_workman.h`.

Reply:
That's pretty much it, I've only changed 3 characters and everything else is the same as existing workman